Bear:
  joints_update:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  front_right_leg_controller:
    type: "position_controllers/JointTrajectoryController"
    joints:
      - Femur_FR
      - Tibia_FR

    constaints:
      goal_time: 0.5
      Femur_FR: {goal: 1}
      Tibia_FR: {goal: 1}

    stop_trajectory_duration: 0.2
    state_publish_rate: 50
